src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -227,6 +227,33 @@ show_device_conf(xrt_core::device* device)
   std::cout << std::flush;
 }
 
+/*
+ * helper function for option:purge
+ * remove the daemon config file
+ */
+static void 
+remove_daemon_config()
+{
+  XBU::sudo_or_throw("Removing daemon configuration file requires sudo");
+  try {
+    /* 
```

Comment:
I like comments and I am delighted to see them in the code base, so it hurts me to ask to remove this comment.  It is not needed.

Reply:
Sure @rozumx . Removed the comment.